lib/gelf/logger.rb

```diff
@@ -5,7 +5,7 @@ module LoggerCompatibility
     attr_accessor :formatter
 
     # Use it like Logger#add... or better not to use at all.
-    def add(level, message = nil, progname = nil)
+    def add(level, message = nil, progname = nil, &block)
```

Comment:
But what is the point of passing a block if it is never called? Is it required for compatibility?